VOTER_RTCM-firmware/build-files/src/Voter.c

```diff
@@ -1926,112 +1885,112 @@ void __attribute__((interrupt, auto_psv)) _MathError(void)
 
 int myfgets(char *buffer, unsigned int len);
 
-#if defined(SMT_BOARD)
-
+#ifdef SMT_BOARD
 ROM WORD ledmask[] = {0x1000,0x800,0x400,0x2000};
 
-void SetLED(BYTE led,BOOL val)
-{
+void SetLED(BYTE led,BOOL val) {
 	LATB &= ~ledmask[led];
-	
-	if (!val) LATB |= ledmask[led];
+	if (!val) {
+		LATB |= ledmask[led];
+	}
 }
 	
-void ToggleLED(BYTE led)
-{
+void ToggleLED(BYTE led) {
 	LATB ^= ledmask[led];
 }
 	
-void SetPTT(BOOL val)
-{
+void SetPTT(BOOL val) {
 	_LATB4 = val;	
 }
 	
-void SetAudioSrc(void)
-{
+void SetAudioSrc(void) {

```

Comment:
The "standard" we are using in Asterisk, functions have the { on a new line, if/while have { on the same line.
It's not a "big deal", just pointing out the way Asterisk base is.